modules/age.nix

```diff
@@ -63,6 +65,7 @@ with lib; let
 
   installSecret = secretType: ''
     ${setTruePath secretType}
+    export PATH="${pkgs.gnugrep}/bin:${pkgs.coreutils}/bin:@out@/sw/bin:/usr/bin:/bin:/usr/sbin:/sbin${pluginPaths}"
```

Comment:
This looks weird to me. I imagine grep and coreutils were for poking around, but I don't think we want to allow the impure paths `/usr/bin`, `/bin`, `/usr/sbin`, and `/sbin` to affect the PATH at this point.

We might want to wrap the `ageBin` binary to add these plugins to its PATH without affecting the PATH for the rest of the script.

Reply:
Yeah, I honestly doubt most of those path elements are necessary. I started out adjusting the existing PATH as I'm used to in shell (e.g. `PATH="${PATH}:something"`) but since that doesn't play very nicely with Nix's variable substitution I just copied the PATH from another script somewhere else in de module.

I'll adjust it to be more precise.